models/_sector/dex/trades/dex_trades_beta.sql

```diff
@@ -0,0 +1,31 @@
+{{ config(
+    tags=['dunesql'],
+    schema = 'dex',
+    alias ='trades_beta',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'project', 'version', 'tx_hash', 'evt_index']
+    )
+}}
+
+-- (blockchain, project, project_version, model)
+{% set base_models = [
+     ('ethereum',   'defiswap',   '1',    ref('defiswap_ethereum_base_trades'))
+    ,('ethereum',   'uniswap',    '1',    ref('uniswap_ethereum_v1_base_trades'))
+    ,('ethereum',   'uniswap',    '2',    ref('uniswap_ethereum_v2_base_trades'))
+    ,('ethereum',   'uniswap',    '3',    ref('uniswap_ethereum_v3_base_trades'))
+] %}
+
+
+-- macros/models/sector/dex
+{{
+    dex_enrich_trades(
+        blockchain = 'ethereum'
```

Comment:
how can i set this to:
`blockchain = base_models[0]`?

Reply:
then we can control blockchain name in the above variable, keep it model specific, then we can be cross-chain here

Reply:
We can remove `blockchain` parameter in macro dex_enrich_trades() to implement this.
i.e.
```
{% macro dex_enrich_trades(
    models=[],
    transactions_model=null,
    tokens_erc20_model=null,
    prices_model=null
    )
%}

WITH base_union AS (
    {% for dex_model in models %}
    SELECT
        '{{ dex_model[0] }}' as blockchain,
        '{{ dex_model[1] }}' as project,
        '{{ dex_model[2] }}' as version,
...
```